### PR TITLE
Fix freecodecamp oop content

### PR DESCRIPTION
**Pull request by Ngoakor12**
Related issues: [please specify]

## Description:
Updated freecodecamp oop on which section to complete.
What are you up to? Fill us in :)

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: content/topics/free-code-camp/javascript-data-structures-and-algorithms/7-object-oriented-programming/_index.md [hunk 1]
```diff
@@ -2,21 +2,21 @@
...
-title: FreeCodeCamp - Object Orientated Programming
+  - javascript
+  - oop
+title: FreeCodeCamp - Object Oriented Programming
 ---
 
 Make an account here if you haven't already: https://www.freecodecamp.org/learn
 
-Go to the Menu and select Curriculum, under "JavaScript Algorithms and Data Structures," complete the sections "Build JavaScript Objects" to "Accessing Nested Objects".
+Under "JavaScript Algorithms and Data Structures," complete the all challenges under "Object Oriented Programming".
```
Comment (FaithMo) [inline content/topics/free-code-camp/javascript-data-structures-and-algorithms/7-object-oriented-programming/_index.md]: I think this "Go to the Menu and select Curriculum, under " is still essential to make navigation easier for some learners who might not know where the stuff is on the website

**Review by FaithMo — CHANGES_REQUESTED** [review]

File: content/topics/free-code-camp/javascript-data-structures-and-algorithms/7-object-oriented-programming/_index.md (hunk 1 above)
Comment (Ngoakor12) [reply content/topics/free-code-camp/javascript-data-structures-and-algorithms/7-object-oriented-programming/_index.md]: You're automatically navigated to the curriculum page when you first sign in. But I guess it doesn't hurt to be specific. Will change it back.

**Review by FaithMo — APPROVED** [review]

**Review by TheMaja — CHANGES_REQUESTED** [review]
Is it ? `complete the all challenges under "Object Oriented Programming` 
 or 
`complete all challenges under "Object Oriented Programming`

**Review by ry-oc — APPROVED** [review]
looks good :)

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**